packages/toolkit/src/view/catalog/components/lib/helpers.tsx

```diff
@@ -219,3 +219,40 @@ export const formatDate = (dateString: string): string => {
   const date = new Date(dateString);
   return date.toLocaleString();
 };
+
+export const convertTagsToArray = (
+  tags: string | string[] | undefined,
+): string[] => {
+  if (Array.isArray(tags)) return tags;
+  if (typeof tags === "string") {
+    return tags
+      .split(",")
+      .map((tag) => tag.trim())
+      .filter((tag) => tag !== "");
```

Comment:
You do the trim and check for empty for the string, but let the array through. I think it should do the same checks for both: string and array.